src/content/development/contribution-guide/_index.en.md

```diff
@@ -58,7 +58,34 @@ To be expanded
 
 #### Pre-Planning
 
-To be expanded
+The week before the release is dedicated to planning the next release.
```

Comment:
```suggestion
The week before a new release cycle starts is dedicated to planning the next release.
```

Reply:
To clarify that the release being discussed is the same in both instances:

```suggestion
The week before a new release cycle starts is dedicated to planning the release.
```